plugins/available/dirs.plugin.bash

```diff
@@ -80,6 +80,8 @@ S () {				# SAVE a BOOKMARK
     about 'save a bookmark'
     group 'dirs'
```

Comment:
It would be great if you could update the documentation to add the param and an example. Take a look at the `G` function in the same file.